src/prediction/predictCarbon.ts

```diff
@@ -76,18 +57,18 @@ export async function predictCarbon(
   molecule: Molecule,
   options: PredictCarbonOptions = {},
 ): Promise<Prediction1D> {
+  if (molecule.getAllAtoms() === 0) {
+    throw new Error(
+      'Molecule should contains at least one atom for prediction',
```

Comment:
```suggestion
      'Molecule must contain at least one atom for prediction',
```

Since this is Carbon prediction, doesn't it need at least one carbon?

Reply:
if the there is not a carbon in the molecule, the prediction is an empty array of signals. but if it is an empty molecule, the web service throws an error of `empty molecule`

@lpatiny prefer to return an empty prediction like:

```
{
      molfile: molecule.toMolfile(),
      nucleus: '13C',
      diaIDs: [],
      joinedSignals: [],
      signals: [],
      ranges: [],
      molecule,
    }
    ```